City-of-Helsinki/drupal-helfi-etusivu
# UHF-8618

# [UHF-8618](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8618)

Removed old throwaway news migration module. (helfi_migration)
Updated composer.json's project information.
Updated environment specific contrib-modules to D10 compatible version.
- drupal/views_rss rc2 -> rc3
  - [Changes](https://www.drupal.org/project/views_rss/releases/8.x-2.0-rc3)
- Contains also redis update altho the 1.6 version was already D10 compatible

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-8618`
  * `make fresh`
* Run `make drush-cr`

## How to test
- After site has been set up, go to status page. It should load ok.  `/fi/admin/reports/status`
- Go to RSS feed, it should show a bunch of items.  `/fi/uutiset/rss`


[UHF-8618]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8618?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
